g2pf/client.py

```diff
@@ -5,14 +5,40 @@
 class ClientManager:
 
     def __init__(self, args):
-        pass
-
-    def add_client(self, datasets=None, featuresets=None,
-                   phenotypeassociationsets=None):
+        self.clientList=[]
+        self.restriction_mapping_datasets={}
+        self.restriction_mapping_featuresets={}
+        self.restriction_mapping_phenotypeassociationsets={}
+        
+    def add_client(self, datasets=None, featuresets=None, phenotypeassociationsets=None):
         """Add a g2p client to manager. Optionally restrict to specified
         datasets, featuresets, and phenotypeassociatonsets."""
         pass
-
+    
+    def add_http_client(self, url_prefix, datasets=None, featuresets=None, phenotypeassociationsets=None):
```

Comment:
I like your addition of http and local clients. For this project I was envisioning only http, but it makes sense that a user may also have local clients.

I recommend that you also remove L13-17, as we do not need an ambiguous `add_client` as well.